.github/workflows/ci.yaml

```diff
@@ -21,4 +21,4 @@ jobs:
         env:
           TEST_POD_INDEX_NAME: ${{ secrets.TEST_POD_INDEX_NAME }}
           TEST_SERVERLESS_INDEX_NAME: ${{ secrets.TEST_SERVERLESS_INDEX_NAME }}
-          API_KEY: ${{ secrets.API_KEY }}
\ No newline at end of file
+          INTEGRATION_PINECONE_API_KEY: ${{ secrets.API_KEY }}
```

Comment:
This doesn't seem to match up with `.env.example` above

Reply:
I split things out a bit and there are 2 environment keys for setting the API key:
- `PINECONE_API_KEY`
- `INTEGRATION_PINECONE_API_KEY`

Let me know if that's too confusing. Trying to test properly while one env value was used in both situations (the client vs. the tests) felt trickier than just having explicit values for it.